common/network-common/pom.xml

```diff
@@ -45,6 +45,22 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
```

Comment:
Oh maybe I misunderstood, why not you mean move it all there?  I think the only reason was it was a utility class that could be used by others. Both network-shuffle and network-yarn use it now.